package/README.md

```diff
@@ -54,6 +54,7 @@ export default defineConfig({
 astroHashnode({
   hashnodeURL: 'astroplayground.hashnode.dev', // Your hashnode URL
   landingPage: true, // Lets you disable the default landing page!
+  disableViewTransitions: false, // Lets you disable the Default included ViewTransitions.
```

Comment:
There's probably a correct way of doing this semantically, but I would just have this option as `viewTransitions: boolean` and have it true by default

Reply:
This is just showing the default as an example in the readme!

Reply:
Ah I see! Even still I'd consider changing the name